brave/brave-core
# Implement needs browser update to see ads alerts.

Show need to update browser alert on NTP and on rewards settings page.


Resolves https://github.com/brave/brave-browser/issues/21388

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Make any of the Ads requests (to `ads-serve.brave.com` or `ads-static.brave.com`) return code 426 `HTTP Upgrade Required`.
- Check that following alerts appear: https://github.com/brave/brave-core/pull/13377#issuecomment-1131600632
